docs/source/prereqs.rst

```diff
@@ -77,47 +76,6 @@ following command from a terminal prompt:
 
 .. _Go:
 
-Go Programming Language
```

Comment:
Instead of removing all the Go info, it should be moved to devenv topic for Fabric developers to see.

Reply:
Done.

Reply:
Matt said:
There's actually not much to say any more about go. The prereq is already called out in the section before this and there should be no explicit setup required to build. It really should be as simple as cloning the repo and using make to build and test.

so, honestly, I think it can probably all go.